lib/registry.js

```diff
@@ -14,7 +14,11 @@ var Registry = module.exports = function(opts){
   if (opts.db) {
     this.db = opts.db;
   } else {
-    this.location = path.join(process.cwd(), opts.path);
```

Comment:
If we changed this to be easier to override work for you @kyork-cl ? I'd prefer not to break backwards compatibility with node.

Something like this?

```
this.location = opts.path;
```